### ISSUE TITLE
Precision and Recall

### ISSUE THREAD

**Issue by endijsb** [jensleitloff/CNN-Sentinel#16]
Thanks for the code! As I just start to work with machine learning models, then I hope you can help me with your code. I tuned a RGB model and after that made a confusion matrix and classification report, for EuroSAT imagery (All classes) a got a low metrics (precision, recall and f1-score ranged within 0.09 to 0,12), however other parameters were ok (loss: 0.0208 - categorical_accuracy: 0.9924 - val_loss: 0.0805 - val_categorical_accuracy: 0.9764). For main RGB set (four main classes in landscape (deciduous forest, coniferous forest, agriculture land and lakes) the accuracy metrics ranged from 0.2 to 0.3, also low, but at the same time categorical accuracies were high. I wondering why I got such low accuracy metrics and is there any solution to increase them.

**Comment by jensleitloff** [comment]
This is hard to answer, without seeing the code. I linked some pretrained models in a previous Issue #2. You could test those.

Generally, the repository is not intended for productive use. I strongly recommend a segmentation approach (instead of classification) with corresponding datasets such as [SEN12MS](https://github.com/schmitt-muc/SEN12MS) or [BigEarthNet](https://bigearth.net/).

**Comment by endijsb** [comment]
Ok, i will check Issue 2. During the model development what were the accuracy metrics you were achieved for finetuneRGB using vgg? In addition, maybe it possible,  that you update code with classification report?

**Comment by jensleitloff** [comment]
The initial code is four years old and only a simple introduction. If you want, you can fork the repository or make a pull request. I will not add a classification report. 
I can't remember the metrics, but you should be able to restore them using the pretrained models.
